### DISCUSSION TITLE
Projects: Back to Overview

### DISCUSSION THREAD

**Discussion by SebastianThomas** [community/community#15094] [Projects and Issues] [answered]
Please add a "Back to Overview" button (from `/orgs/ORG_NAME/projects/X` back to `/orgs/ORG_NAME/projects/X`), or leave the horizontal navigation bar on the Project-site (
![image](https://user-images.githubusercontent.com/54369369/163726561-8d8625f0-27e0-41c9-9689-244ca707834c.png)
)

**Comment by ARolek** [comment]
Yes! I find myself looking for this every time I'm in a project.

**Comment by th0rgall** [comment, marked as answer]
Agreed on both points! Jumping from one project to another is really a pain.

@SebastianThomas did you mean?: 
> from `/orgs/ORG_NAME/projects/X` back to `/orgs/ORG_NAME/projects` ~~/X~~

It would also be great to list specific projects in the Cmd + K search view.

----
Related issues were collected here https://github.com/github/feedback/discussions/10448, go upvote that one.
